KrysiaBartek2024/source/drury.tex

```diff
@@ -10,11 +10,61 @@
 \begin{document}
 \maketitle
 
+%%% ID: 9
 %%% PRIORITY: 90
 %%% TITLE: Drury
 %%% SYSTEM BEGIN %%%
 
-TBD
+{\textbf{OFF}} in competition
+
+\subsubsection*{P -- 1\majs \\ ?}
+\begin{itemize}
+    \item 1\nt = 8-11, no fit
+    \item 2\clubs = 4-fit mixed raise (7)8-10\dps OR 3-fit (9)10-11\dps
+    \item 2\majs = 3-fit, 4-8\dps
+    \item 2\anysuit{x} = (9)10, solid 5\anysuit{x}
+    \item 3\clubs = (9)10, \inv, 6\clubs
+    \item 3\anysuit{x} = 4-fit, solid 5\anysuit{x}
+    \item 2\nt = 4-fit, solid 5\clubs
+    \item 3\majs = 5-fit 4-6\dps (or 4 with shortness)
+    \item 3\nt = Two Tiered Splinters 4+\majs (unspecified singleton, (10)11\dps)
```

Comment:
1H - 3NT
4C - (ups mam singla pik)

Chyba że się zgadzamy żeby to rozumieć przez domysł, ale komentarz dla potomnych by się imo przydał

Reply:
??